versions.props

```diff
@@ -12,7 +12,7 @@ com.googlecode.json-simple:json-simple = 1.1.1
 com.jayway.awaitility:awaitility = 1.6.5
 com.palantir.config.crypto:encrypted-config-value-module = 1.0.0
 com.palantir.remoting:* = 0.13.0
-com.palantir.remoting1:* = 1.0.3
+com.palantir.remoting1:* = 1.1.0
```

Comment:
The upstream product depends on 1.1.0 and fails on version conflict, so we're in lockstep with them.